### PR TITLE
docs(bodiless-documentation): Update docs regarding Tailwind configuration files

### LINKED ISSUE #1535: [DOC] Update Tailwind Configuration File Information
## User Story
AS AN end user of the Bodiless documentation,
I WANT to understand how to the Tailwind configuration works in Bodiless,
SO THAT, I can properly apply styling to my site.

## Acceptance Criteria
1. The Documentation is updated to include how a user can configure their sites'/packages' Tailwind files.

### PR DESCRIPTION
**Pull request by misterjones**
## Changes

* Update documentation to reflect changes made to how Tailwind configuration files work in Bodiless.
* Update external links in Tailwind Guide to open in new tab.

### Documents:

* #1277 

---

Resolves #1535

### CODE REVIEW

**Review by hvanyo — APPROVED** [review]

**Comment by hvanyo** [comment]
@konst8  please review.
@misterjones  don't worry about the semantic pull request failing -- we can address when we squash & merge.

File: packages/bodiless-documentation/doc/Development/Guides/BuildingSites/Typography/TailwindGuide.md [hunk 1]
```diff
@@ -42,27 +42,32 @@ always have precedence**.
 
 01. Add a `tailwind.config.js` file to the root of the package.
     ```js
-    const plugin = require('tailwindcss/plugin');
+    import { getPackageTailwindConfig } from '@bodiless/fclasses';
 
-    module.exports = {
-      content: [
+    const twConfig = {
+      purge: [
```
Comment (konst8) [inline packages/bodiless-documentation/doc/Development/Guides/BuildingSites/Typography/TailwindGuide.md:48]: @misterjones I believe the "content" was correct:
https://github.com/johnsonandjohnson/Bodiless-JS/blob/chore/upgrade-to-gatsby-4-and-tw-3/packages/fclasses/src/tailwindcss/mergeConfigs.ts#L20
@hvanyo

Comment (hvanyo) [reply packages/bodiless-documentation/doc/Development/Guides/BuildingSites/Typography/TailwindGuide.md:48]: correct on tw upgrade it should be content:

Comment (misterjones) [reply packages/bodiless-documentation/doc/Development/Guides/BuildingSites/Typography/TailwindGuide.md]: ✅  Fixed

**Review by konst8 — APPROVED** [review]

**Pull request merged**